analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RoslynSymbolicExecution.cs

```diff
@@ -84,7 +84,7 @@ private IEnumerable<ExplodedNode> ProcessBranching(ExplodedNode node)
         {
```

Comment:
NITPICK: all the constructor invocations for `ExplodedNode` in this method can be simplified to use target typed `new`.

Reply:
Mosty done in the other PR :)